pkg/xds/bootstrap/generator.go

```diff
@@ -101,6 +106,22 @@ func (b *bootstrapGenerator) generateFor(proxyId core_xds.ProxyId, dataplane *co
 	return b.configForParameters(params)
 }
 
+func (b *bootstrapGenerator) verifyAdminPort(adminPort uint32, dataplane *core_mesh.DataplaneResource) error {
+	for _, inbound := range dataplane.Spec.Networking.Inbound {
+		if adminPort == inbound.Port {
+			return errors.Errorf("Resource precondition failed: Port %d requested as both admin and inbound port.", adminPort)
+		}
+	}
```

Comment:
inbound can have an address specified therefore even with the same port there won't be a clash.
For inbounds, if address is not specified, then the address from networking is taken.

Reply:
Got it, thanks for the clarification. Fixing this.